packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -281,7 +281,17 @@ function VM-Install-Shortcut{
     )
     $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
     $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+    $extension = [System.IO.Path]::GetExtension($executablePath)
 
+    # If the shortcut is for a .py, we should set the shortcut icon to be Python (requires Python to be installed)
+    if ($extension -eq ".py") {
+        $pythonPath = (Get-Command python).Source
```

Comment:
according to your comment in https://github.com/mandiant/VM-Packages/pull/855#discussion_r1463383496, we should execute .py files with Python instead of changing the icon, do we then need this code?

I think the only case where that could make sense is diaphora, but it is a very special case, so I would rather have that in the package that here if it is the only reason.

Reply:
While I agree that we -should- execute python files with Python in the `$executablePath`, this was more of a fallback option.

I can remove it if we notate somewhere in the documentation that python tools should use Python for it's executable path and I'll go through each of our current Python tools list and make the changes. :)

Reply:
please document this in the wiki, ideally this week as I'll be restructuring the wiki next week to propose a better version of the documentation. Do not worry much where the best place for this is, as I can move it next week. :wink:

Reply:
Added!